cmake/EthDependencies.cmake

```diff
@@ -29,8 +48,12 @@ if (DEFINED MSVC)
 	endif()
 	set (CMAKE_PREFIX_PATH ${ETH_DEPENDENCY_INSTALL_DIR} ${CMAKE_PREFIX_PATH})
 
-	# TODO it windows SDK is NOT FOUND, throw ERROR
+	# TODO if windows SDK is NOT FOUND, throw ERROR
 	# from https://github.com/rpavlik/cmake-modules/blob/master/FindWindowsSDK.cmake
+	#
+	# TODO What are we using WindowsSDK for specifically?  Do we need to do that, or
```

Comment:
This is probably not needed. I think we only need the C++ compiler and the runtime lib, but cmake can handle that itself. IIRC I was able to build without searching for Windows SDK.